src/_data/about.json

```diff
@@ -1,7 +1,10 @@
 {
   "title": "About Me",
-  "bioHeader": "I am <mark class='highlight'>based in<a href='https://en.wikipedia.org/wiki/indonesia' target='_blank' rel='noopener noreferrer'> Indonesia<span class='new-tab-warning'> Open a new tab </span>.</a></mark>",
-  "bio": "I am currently Majoring in Computer Science at<a href='https://ukim.ac.id' target='_blank' rel='noopener noreferrer'><mark class='highlight'>Christian University of Indonesia, Moluccas.</mark><span class='new-tab-warning'> Open a new tab </span></a><br/>I am also a <i>Certified Fiber Optic Technician</i> since 2017 by <a href='http://apjatel.id/' target='_blank' rel='noopener noreferrer'><mark class='highlight'>APJTEL.</mark><span class='new-tab-warning'> Open a new tab </span></a></div>",
+  "bioHeader": "I am ==based in [Indonesia.](https://en.wikipedia.org/wiki/indonesia){target=_blank rel=\"noopener noreferrer\"}=={.highlight}",
+  "bio": [
+    "I am currently Majoring in Computer Science at ==[Indonesian Christian University in the Moluccas](https://ukim.ac.id){target=_blank rel=\"noopener noreferrer\"}=={.highlight}",
+    " I am also a <i>Certified Fiber Optic Technician</i> since 2017 by ==[APJTEL](http://apjatel.id/){target=_blank rel=\"noopener noreferrer\"}=={.highlight}"
+  ],

```

Comment:
With the new formatting, I guess, there should be a documentation on how to do it, or at least explain step by step on how to properly add or modify the syntax.